images/entrypoint.sh

```diff
@@ -380,6 +380,9 @@ generateMultusConf
 
 # ---------------------- end Generate "00-multus.conf".
 
+# remove "00-multus.conf" on close.
+trap "rm -f $CNI_CONF_DIR/00-multus.conf" EXIT
```

Comment:
IMO This should be delete only if it was created by the entrypoint.

i.e: `"$MULTUS_CONF_FILE" == "auto"`

also i wonder if it should depend on `MULTUS_CLEANUP_CONFIG_ON_EXIT`